js/muzima.js

```diff
@@ -163,6 +163,9 @@ $(document).ready(function () {
             validForm = validForm && $.fn.customValidationCheck();
         }
         if (validForm) {
+	    $(this).find('input:text').each(function(){
```

Comment:
Is this intentional? Because if we need this process, then we need to put inside muzima.js in the muzima-android project. Adding this comment and merging this PR.